.github/workflows/ci.yml

```
@@ -22,10 +22,10 @@ jobs:
           - macos-latest
           - ubuntu-latest
         ocaml-compiler:
-          - ocaml-base-compiler.4.14.1
+          - “4.14”
```

Just passing by, could not resist :)
```suggestion
          - "4.14"
```